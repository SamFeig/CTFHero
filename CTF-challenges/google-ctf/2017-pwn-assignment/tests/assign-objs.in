a.b.c=1
d=a.b
d
c=1
c=c
c
e.h=12345
e.h=c
e
a.c.d=5
a.e="2"
a
a.f.e.g=a.f
a
