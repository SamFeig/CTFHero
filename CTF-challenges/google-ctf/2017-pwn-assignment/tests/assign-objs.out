> > > {
  c: 1
}
> > > 1
> > > {
  h: 1
}
> > > {
  e: "2"
  c: {
    d: 5
  }
  b: {
    c: 1
  }
}
> > {
  f: {
    e: {
      g: { ... }
    }
  }
  e: "2"
  c: {
    d: 5
  }
  b: {
    c: 1
  }
}
> Failed reading
